Add Fetch and Move PG AST nodes

mulias opened the pull request:
Add https://www.postgresql.org/docs/current/sql-fetch.html and https://www.postgresql.org/docs/current/sql-move.html which are related

mulias committed:
Add Fetch and Move PG AST nodes

mulias committed:
Fix missing exports for SetConfigurationParameter and SetTimeZone

ClaireNeveu commented on packages/sij-dialect-postgresql/src/ast/util.ts:
```diff
@@ -13,6 +17,83 @@ type PgUtil =
+  | Absolute
+  | Relative
+  | 'All'
+  | 'Forward'
```
I would make these duplicate cases into one case with the `count` as `NumLit | null`.

mulias replied:
To be more general, would you prefer if this list had the minimal number of options necessary to cover all cases? For example `Next`,  and `Forward`, are equivalent to `Forward 1` or `Relative 1`. I think I could get rid of `Next`, `Prior`, `First`, `Last`,  `All`, `Forward`, and `Backward`. That would leave `Absolute count`, `Relative count`, `Forward count`, `Backward count`, `ForwardAll`, and `BackwardAll`.

mulias replied:
It's not clear to me which you're seeing as the duplicate cases.

mulias replied:
Oh I see it, we could actually do `count: NumLit | 'All' | null`

mulias committed:
Simplify direction AST for Fetch and Move

Replace `'Forward'` with `Forward({count: 1})` and `ForwardAll` with
`Forward({count: 'All'})`. Ditto with `Backward`.
